coderedcms/tests/test_templatetags.py

```diff
@@ -5,7 +5,7 @@
 
 
 django_engine = engines['django']
-html_id_re = re.compile("^[A-Za-z][A-Za-z0-9_:\.-]*$")
+html_id_re = re.compile("^[A-Za-z][A-Za-z0-9_:\.-]*$")  # noqa: W605
```

Comment:
This should probably use a raw string (`r""` ) to avoid the escape sequence confusion.